platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_structuring_builds.adoc

```diff
@@ -506,14 +506,16 @@ You can also include projects located in subdirectories using hierarchical paths
 This allows you to organize large builds containing many projects to improve comprehensibility, versus having all your projects live at the root of the build.
 For instance, if project `search` was located in a subdirectory named `features`, itself located in a subdirectory named `subs`, you could call `include(":subs:features:search")` to include it.
 
+NOTE: You should typically keep your _logical_ project paths and _physical_ project locations aligned to avoid confusion.  Don't include a project named `:subs:features:search` and locate it at `features/ui/search`, as this will lead to confusion about the location of the project.
+
 However, without further configuration, Gradle will create empty projects for every element of the path.
 In the example above, Gradle will create a project named `:subs`, a project named `:subs:features`, and a project named `:subs:features:search`.
 This probably isn't intended, as you likely only want to include the `search` project.
 Additional projects - even if empty - will slow down your build, clutter reporting, and make your build harder to understand.
 
 To avoid this, be sure to explicitly set the link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:projectDir[Project.projectDir] property of any projects included in nested directories: `project(':my-web-module').projectDir = file("subs/web/my-web-module"`).
 
-As ab added benefit, this will allow you to reference the project by the shorter name `:my-web-module` instead of the full logical path `:subs:web:my-web-module`.
+As an added benefit, this will allow you to reference the project by the shorter name `:my-web-module` instead of the full logical path `:subs:web:my-web-module`.
```

Comment:
I thought we wanted to either get rid of this sentence altogether or add a disclaimer about how it's still better to use the full format `:subs:web:my-web-module` so you don't forget "where" the project is located.